.github/workflows/actions_build.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 on:
   push:
     branches:
-      - main
+      - "**"
```

Comment:
Question) Is this equivalent to if we just remove these two lines altogether?

Reply:
I tried this setting on my forked reop first but it didn't seem to work.
```yaml
on:
  push:
    tags-ignore:
      # The release versions will be verified by 'publish-release.yml'
      - armeria-*
  pull_request:
  merge_group:
```